### ISSUE TITLE
Support for touchpadrc

### ISSUE THREAD

**Issue by MrQubo** [nix-community/plasma-manager#324]
I think `~/.config/touchpadrc` is only used on X11. And probably only with synaptics drivers. It stores touchpad settings.

**Comment by HeitorAugustoLN** [comment]
I will look into it.

**Comment by MrQubo** [comment]
I can work on this, is there any "Getting started for developers"?
I assume I would need to add module options and generate config file from that. Add parsing to rc2nix too. Anything else?

**Comment by MrQubo** [comment]
Not sure where to add it though. `options.programs.plasma.input.synaptics-touchpad`?

**Comment by HeitorAugustoLN** [comment]
> Not sure where to add it though. `options.programs.plasma.input.synaptics-touchpad`?

Looks good to me.

**Comment by HeitorAugustoLN** [comment]
> I can work on this, is there any "Getting started for developers"? I assume I would need to add module options and generate config file from that. Add parsing to rc2nix too. Anything else?

No, we don't have that. But it shouldn't be that hard to figure it out. It is basically the same thing as doing other nixos modules, the only difference is that ours use our script as a backend. You can open a draft PR and I can help you if you need.
